org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/client/CodeLensKindCapabilities.java

```diff
@@ -13,8 +13,7 @@
 
 import java.util.List;
 
-import org.eclipse.xtext.xbase.lib.Pure;
-import org.eclipse.xtext.xbase.lib.util.ToStringBuilder;
+import org.eclipse.lsp4j.jsonrpc.util.ToStringBuilder;

```

Comment:
This was not available in lsp4j 0.20.x

Reply:
Hm... do one really needs this helper class here? It really seems not suing all the facy stuff here any maybe a standard java StingBuilder would suffice?

Reply:
It can be inlined for sure.